src/platform/EFR32/KeyValueStoreManagerImpl.cpp

```diff
@@ -146,7 +144,6 @@ CHIP_ERROR KeyValueStoreManagerImpl::_Get(const char * key, void * value, size_t
 CHIP_ERROR KeyValueStoreManagerImpl::_Put(const char * key, const void * value, size_t value_size)
 {
     VerifyOrReturnError(key != nullptr, CHIP_ERROR_INVALID_ARGUMENT);
-    VerifyOrReturnError(value != nullptr, CHIP_ERROR_INVALID_ARGUMENT);
```

Comment:
Value can only be nullptr if value_size is 0

Reply:
_Put then calls EFR32Config::WriteConfigValueBin which does the correct check :)